### PR TITLE
#826-Web page is not being navigated sequentially with the Tab key at…

### PR DESCRIPTION
**Pull request by ramdev1980**
… Navigation screen

### CODE REVIEW

File: WebUI/war/plugins/PercListEditorWidget.js [hunk 1]
```diff
@@ -46,15 +46,15 @@
             '    </div>' +
             '' +
             '    <div id="perc-ui-listedit-listitemfield">' +
-            '        <input id="perc-ui-permission-usernamefield" title="'+I18N.message("perc.ui.rolePropsDialog.title@Enter a role")+'" name="perc-ui-permission-usernamefield"/>' +
-            '        <div id="perc-ui-permission-plusbutton" role="button" title="'+I18N.message("perc.ui.rolePropsDialog.title@Add User")+'"></div>' +
+            '        <input id="perc-ui-permission-usernamefield" tabindex="30" title="'+I18N.message("perc.ui.rolePropsDialog.title@Enter a role")+'" name="perc-ui-permission-usernamefield"/>' +
+            '        <div id="perc-ui-permission-plusbutton" tabindex="30" role="button" title="'+I18N.message("perc.ui.rolePropsDialog.title@Add User")+'"></div>' +
```
Comment (natechadwick) [inline WebUI/war/plugins/PercListEditorWidget.js]: @ramdev1980 Why do they all have 30 as the tabindex?

Comment (natechadwick) [reply WebUI/war/plugins/PercListEditorWidget.js]: Holding this from 8.1.3 as one of those files is used in a number of screens.  We need to make sure we test that the issue is resolved on all of the use cases.

Comment (natechadwick) [reply WebUI/war/plugins/PercListEditorWidget.js]: @ramdev1980  Can we use tabindex 0 on these?

Comment (ramdev1980) [reply WebUI/war/plugins/PercListEditorWidget.js]: @natechadwick  I have committed the changes. Please review.

**Review by sixartifact260 — APPROVED** [review]

**Review by sixartifact260 — APPROVED** [review]

**Review by natechadwick — APPROVED** [review]

**Pull request merged**